CHANGES.md

```diff
@@ -24,6 +24,32 @@ OpenSSL 3.0
 
 ### Changes between 1.1.1 and 3.0 [xx XXX xxxx] ###
 
+
+ * The main project documents (README, NEWS, CHANGES, INSTALL, SUPPORT)
+   have been converted to Markdown with the goal to produce documents
+   which not only look pretty when viewed online in the browser, but
+   remain well readable inside a plain text editor.
+
+   To achieve this goal, a 'minimalistic' Markdown style has been applied
+   which avoids formatting elements that interfere too mcuh with the
```

Comment:
mcuh -> much

Reply:
Thanks, I noticed that immediately after pushing it myself. Please note that I pushed two fixups already.